### PR TITLE
Close #3765 Change top-level Quickstart toolbar link to an internal page instead of an external page.

### LINKED ISSUE #3765: Link in admin UI goes to external site
## Problem/Motivation
I want to open the Quickstart 2.x.x menu in the admin UI in a new tab.

### Describe the bug
The admin menu goes to a different page rather than open new tab list.

### To Reproduce

1. Log into a website
2. Right click on "Quickstart 2.x.x"
3. Open in a new tab
4. View quickstart.arizona.edu website

## Proposed resolution
Remove the https://quickstart.arizona.edu link from the admin UI navigation.

### Expected behavior

I expect the menu tabs to appear on the same site I am on.

##  Screenshots
image

### PR DESCRIPTION
**Pull request by trackleft**
## Description
I've updated the link to be an internal route.

We may want to update the bottom of this page https://quickstart.arizona.edu/site-admin/navigating-administration-section

## Related issues
Close #3765 

### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
   - [x] Bug fix
   - [ ] Accessibility, performance, or security improvement
   - [ ] Critical institutional link or brand change
   - [ ] Adding experimental module
   - [ ] Update experimental module
- **Minor release changes**
   - [ ] New feature
   - [ ] Breaking or visual change to existing behavior
   - [ ] Upgrade experimental module to stable
   - [ ] Enable existing module by default or database update
   - [ ] Non-critical brand change
   - [ ] New internal API or API improvement with backwards compatibility
   - [ ] Risky or disruptive cleanup to comply with coding standards
   - [ ] High-risk or disruptive change (requires upgrade path, risks regression, etc.)
- **Other or unknown**
   - [ ] Other or unknown

### Drupal core
- **Patch release changes**
   - [ ] Security update
   - [ ] Patch level release (non-security bug-fix release)
   - [ ] Patch removal that's no longer necessary
- **Minor release changes**
   - [ ] Major or minor level update
- **Other or unknown**
   - [ ] Other or unknown

### Drupal contrib projects
- **Patch release changes**
   - [ ] Security update
   - [ ] Patch or minor level update
   - [ ] Add new module
   - [ ] Patch removal that's no longer necessary
- **Minor release changes**
   - [ ] Major level update
- **Other or unknown**
   - [ ] Other or unknown

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] My change requires release notes.

### CODE REVIEW

**Comment by danahertzberg** [comment]
This is lovely. I'm not sure why the Quickstart 2 link doesn't allow me to open it in a new tab though..

[Screen recording](https://emailarizona-my.sharepoint.com/:v:/g/personal/danahertzberg_arizona_edu/EYk9fjXMY6lLiyayT59Lz3EBTwoxqOEZPnaDoH_T9LfgIA?e=aPF20f&nav=eyJyZWZlcnJhbEluZm8iOnsicmVmZXJyYWxBcHAiOiJTdHJlYW1XZWJBcHAiLCJyZWZlcnJhbFZpZXciOiJTaGFyZURpYWxvZy1MaW5rIiwicmVmZXJyYWxBcHBQbGF0Zm9ybSI6IldlYiIsInJlZmVycmFsTW9kZSI6InZpZXcifX0%3D)

**Comment by danahertzberg** [comment]
Can it link to here, please? `/admin/config/az-quickstart`

File: modules/custom/az_core/az_core.module [hunk 1]
```diff
@@ -44,13 +44,15 @@ function az_core_toolbar() {
     'tab' => [
       '#type' => 'link',
       '#title' => t('Quickstart 2'),
-      '#url' => Url::fromUri('https://quickstart.arizona.edu'),
+      '#url' => Url::fromRoute('<nolink>'),
```
Comment (trackleft) [inline modules/custom/az_core/az_core.module:47]:
```suggestion
      '#url' => Url::fromRoute('az_core.az_quickstart'),
```

**Review by danahertzberg — APPROVED** [review]

**Review by bberndt-uaz — APPROVED** [review]

**Comment by joeparsons** [comment]
> This is a nice change!
> 
> Considering where it goes now (`/admin/config/az-quickstart`), it would be nice if that page included the same links that are within the sub-menu of Quickstart 2.
> 
> It already has Settings, so it'd just need:
> 
> * Releases
> * Documentation
> * Analytics Dashboard
> 
> Not sure if this PR is a fine place to add that or if a separate issue should be made.

Good idea.  Thanks @joshuasosa.  @danahertzberg Is going to make a separate issue for that.

**Comment by danahertzberg** [comment]
@joshuasosa This is a great recommendation, thank you! I've created a new ticket for that here.
https://github.com/az-digital/az_quickstart/issues/3770

**Pull request merged**